doc/builders/packages/dlib.section.md

```diff
@@ -0,0 +1,13 @@
+# DLib
```

Comment:
```suggestion
# DLib {#dlib}
```
We need to add these sections, because we may not have automatic section generation in our markdown tooling and we want to preserve the link URLs.